sensing/laserscan_to_occupancy_grid_map/launch/laserscan_to_occupancy_grid_map.launch.py

```diff
@@ -61,7 +61,7 @@ def add_launch_arg(name: str, default_value=None):
                     "max_height": 2.0,
                     "angle_min": -3.141592,  # -M_PI
                     "angle_max": 3.141592,  # M_PI
-                    "angle_increment": 0.00436332222,  # 0.25*M_PI/180.0
+                    "angle_increment": 0.00349065850,  # 0.20*M_PI/180.0
```

Comment:
I've been using this in dummy perception launch, and I want to make it common with perception launch.

Reply:
change for dummy_perception_publisher_launch to refer perception launch will be at  another PR